ride_share.rb

```diff
@@ -0,0 +1,135 @@
+#Structure of my_company datas looks like this:
+my_company = [
+  {
+    DR0001: 
+    [
+      {
+        date: "3rd Feb 2016",
+        cost: 10,
+        ride_id: "RD0003",
+        rating: 3
+      },
+      {
+        date: "3rd Feb 2016",
+        cost: 30,
+        ride_id:"RD0015",
+        rating: 4
+      },
+      {
+        date: "6th Feb 2016",
+        cost: 45,
+        ride_id: "RD0003",
+        rating: 2
+      }
+    ],
+  },
```

Comment:
Having an array that holds a list of hashes that each have only one element is kind of redundant. Why not just have each of the drivers in the same bigger hash?

For example:
```
drivers = {
  DR0001: [
  ],
  DR002: [
  ]
}
```